bin/silius/src/cli.rs

```diff
@@ -51,6 +51,22 @@ pub struct RpcServiceOpts {
     #[clap(long, value_delimiter=',', default_value = "eth", value_parser = ["eth", "debug", "web3"])]
     pub rpc_api: Vec<String>,
 
+    /// Enables or disables the HTTP RPC.
+    ///
+    /// By default, this option is set to true.
+    /// - To enable: `--http`, `--http true`, or no `http` flag.
+    /// - To disable: `--http false`.
+    #[clap(long, action = Set, num_args=0..=1, default_value = "true", default_missing_value = "true")]
+    pub http: bool,
+
+    /// Enables or disables the WebSocket RPC.
+    ///
+    /// By default, this option is set to true.
+    /// - To enable: `--ws`, `--ws true`, or no `ws` flag.
+    /// - To disable: `--ws false`.
```

Comment:
Actually, it is kind of wired for me to disable with `--ws false`

Reply:
I think we should follow practices from other clients/projects.

https://geth.ethereum.org/docs/interacting-with-geth/rpc#websockets-server

Reply:
thank you for your review.
also i fixed it [`aa0f011` (#195)](https://github.com/Vid201/silius/pull/195/commits/aa0f011003fb9da20e2a10722992e61f5d462014).
We can control true/false by just adding `--http` and `-`-ws` or not.